colir.el

```diff
@@ -37,8 +37,8 @@
   "Select a method to compose two color channels."
   :group 'ivy
   :type '(radio (function-item colir-compose-alpha)
-                (function-item colir-compose-overlay)
-                (function-item colir-compose-soft-light)))
+          (function-item colir-compose-overlay)
+          (function-item colir-compose-soft-light)))
```

Comment:
If you really must reindent this, then please place this and other first `function-item`s on their own line.

Reply:
Done, in all appropriate places.